recipe/meta.yaml

```diff
@@ -30,15 +32,10 @@ requirements:
     - pango
     - pixman
     - pyqt
-    - libgcc  # [linux]
-    - libgfortran  # [osx]
```

Comment:
Using the jinja syntax for compiler we get this recipe ready for the compiler migration and ensure that the proper runtimes will be install accordingly.